read-fonts/src/tables/glyf.rs

```diff
@@ -149,14 +152,35 @@ impl<T> Point<T> {
 }
 
 /// Point with an associated on-curve flag in a simple glyph.
-#[derive(Clone, Copy, Debug)]
+///
+/// This type is a simpler representation of the data in the blob.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub struct CurvePoint {
-    /// The point.
-    pub point: Point<i32>,
+    /// X cooordinate.
+    pub x: i16,
+    /// Y cooordinate.
```

Comment:
I'm not sure commenting Point.y as "y coordinate" is anything more than noise

Reply:
I'm seeing similar noise-level comments quite a bit.

Reply:
Agree with Colin that missing public docs look weird and inconsistent. I also think it gives an impression that the library is not well maintained. FWIW, the standard library also has many cases of “noise” doc comments: https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.new